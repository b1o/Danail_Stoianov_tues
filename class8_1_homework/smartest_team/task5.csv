58,21:57
429,19:06
409,17:22
219,03:48
292,21:58
168,13:20
206,16:47
514,05:11
527,21:41
555,19:11
464,16:17
587,10:44
554,17:04
451,04:18
200,06:15
261,11:25
233,11:13
545,01:11
484,13:07
553,22:07
512,06:22
384,13:20
208,18:53
70,00:28
74,08:25
549,03:03
148,00:03
484,21:33
379,07:25
582,20:52
231,01:57
550,13:24
192,11:22
132,23:37
470,05:06
137,15:49
99,13:40
511,19:15
362,09:21
205,01:14
258,23:41
301,10:34
435,23:51
64,02:27
70,12:28
235,03:20
445,09:18
542,11:32
279,06:25
472,10:09
545,15:20
390,06:22
267,05:51
185,11:56
272,13:17
45,03:49
248,01:18
428,02:33
58,02:50
389,20:22
256,23:11
263,19:14
186,04:39
398,17:08
107,04:30
251,04:06
330,07:55
593,05:53
372,04:44
540,02:53
314,22:37
16,05:03
63,15:37
206,09:57
456,13:28
219,01:01
369,21:51
342,22:00
29,21:54
66,17:02
337,17:20
604,03:22
288,13:37
234,19:18
223,18:06
337,13:07
594,20:40
468,18:58
66,18:14
101,05:26
368,06:27
582,16:04
352,22:52
364,06:16
533,09:15
330,05:17
74,15:39
313,01:02
476,03:48
266,15:24
293,15:14
73,12:15
226,13:50
290,09:52
386,21:58
388,11:23
35,06:38
156,07:16
150,17:58
431,22:24
226,09:27
492,02:34
303,16:32
164,11:00
105,11:45
538,07:47
183,10:13
393,21:32
277,14:30
19,23:55
297,08:57
355,00:51
46,10:01
196,18:12
274,18:21
207,03:00
414,23:05
123,12:47
173,01:10
541,10:21
585,11:23
584,21:21
150,12:25
582,08:46
142,20:42
105,07:23
49,03:48
442,17:51
60,05:54
24,21:45
418,09:57
241,20:35
122,20:09
217,20:09
236,04:02
568,13:13
455,09:00
399,11:37
402,03:09
195,23:07
349,21:47
273,19:24
475,09:33
22,03:30
327,00:58
231,07:44
141,17:38
50,10:09
495,15:52
300,05:17
337,18:35
440,11:27
389,07:13
131,22:58
84,12:16
470,09:22
209,11:19
198,05:33
33,01:58
535,07:05
276,03:33
220,08:28
184,14:26
239,03:30
279,22:51
341,16:21
164,06:44
521,09:14
442,03:08
499,14:57
600,14:43
42,17:24
199,16:02
160,21:35
483,17:55
463,23:01
602,06:48
113,15:34
86,11:04
284,11:32
397,01:08
445,19:49
61,08:19
440,07:59
408,00:01
197,03:43
60,07:52
487,13:31
29,16:57
165,06:26
315,05:51
582,05:09
449,10:03
559,17:18
448,21:27
132,12:38
364,11:11
433,14:01
549,20:18
358,04:46
571,18:06
568,13:01
327,13:44
408,19:22
46,01:31
343,13:48
435,03:22
513,07:29
484,23:59
301,07:02
303,23:14
315,14:43
499,16:08
264,12:25
125,19:09
225,03:33
495,01:11
55,06:53
497,18:47
558,11:36
536,01:42
207,05:24
508,04:36
466,02:20
65,14:16
237,03:45
523,05:12
161,16:42
274,05:29
582,14:36
398,16:40
509,15:38
602,20:31
114,15:57
18,23:28
454,13:27
151,20:59
326,04:19
29,07:50
334,22:33
170,23:48
419,21:08
558,03:13
293,21:12
131,11:50
181,15:01
539,10:58
364,05:21
334,17:37
292,17:05
347,05:35
33,19:24
431,03:01
258,21:35
222,05:42
453,17:20
174,01:40
58,10:41
418,00:03
402,05:36
315,22:29
317,11:36
38,15:44
393,01:12
459,01:46
362,03:52
584,05:11
197,23:21
372,04:28
528,02:24
237,12:31
189,19:43
372,07:10
431,01:31
319,03:44
40,12:06
270,23:58
480,02:48
403,21:01
226,05:22
265,05:41
102,06:50
520,17:11
113,01:28
200,19:02
126,01:08
259,03:44
32,11:51
401,10:46
325,12:31
121,05:23
162,21:37
576,18:21
493,09:22
440,09:39
451,18:08
335,00:10
431,00:00
571,11:48
135,19:18
531,11:31
462,12:40
187,12:18
365,03:45
579,14:49
120,02:15
452,02:02
508,01:08
526,19:17
550,01:18
346,15:42
280,11:19
406,18:18
253,04:55
39,20:38
54,19:16
494,16:55
553,08:13
248,07:28
234,02:34
510,08:21
105,09:06
248,16:52
409,22:38
293,00:31
22,08:39
205,09:00
391,16:17
265,21:51
225,03:19
38,23:20
511,20:21
403,11:20
24,02:53
70,10:49
578,09:32
238,01:10
185,07:46
514,19:40
353,21:13
398,01:53
467,03:27
515,23:18
277,05:51
191,04:10
280,06:26
485,15:34
429,10:26
536,07:04
276,14:46
132,08:11
63,19:52
293,06:44
213,13:26
51,23:54
63,14:30
408,12:21
66,01:42
288,16:24
514,21:38
186,15:31
85,03:12
206,17:18
327,05:01
32,22:39
224,15:31
87,03:39
390,15:36
211,14:52
31,18:37
170,07:34
122,20:23
426,10:42
93,10:26
232,03:26
212,06:08
181,22:44
577,11:37
259,18:22
438,07:21
560,00:37
150,20:54
411,06:34
455,03:08
135,19:42
478,23:05
436,01:40
448,07:52
216,04:53
344,01:35
511,10:28
152,17:49
220,07:01
509,19:36
338,14:53
394,16:46
82,01:58
12,10:01
345,10:10
185,23:37
534,06:23
415,03:55
453,00:33
262,18:56
540,20:57
375,13:29
255,16:26
559,10:04
522,02:37
30,21:25
287,16:11
452,14:02
413,05:34
254,16:30
419,18:10
79,00:26
505,00:42
458,08:38
9,08:09
458,11:04
421,16:28
443,12:46
150,05:48
311,22:28
53,09:26
310,13:22
226,08:15
333,08:27
62,21:55
361,14:05
516,05:33
69,09:49
357,12:19
184,08:20
207,08:02
450,20:04
471,09:02
182,21:23
255,20:16
507,22:39
297,01:14
37,08:02
197,03:15
302,11:55
88,04:33
363,07:05
555,06:12
506,06:16
101,13:28
191,04:46
471,06:57
96,13:35
175,07:20
93,11:24
561,03:11
392,04:03
194,07:21
390,04:45
561,19:50
475,02:28
48,18:49
518,04:55
505,00:01
574,15:24
364,04:37
572,18:58
262,01:25
289,09:22
410,06:01
116,15:26
360,00:22
461,11:44
18,13:59
507,22:38
125,00:40
21,23:36
500,21:07
385,15:11
11,08:59
167,06:17
504,09:04
447,08:45
291,12:23
110,00:05
64,07:02
37,17:40
58,17:23
448,09:07
195,13:49
441,09:28
480,17:20
116,03:20
217,15:34
551,09:15
273,16:45
15,01:51
559,08:49
415,15:56
540,06:40
93,05:46
490,16:44
42,06:40
5,18:56
332,04:24
129,12:37
351,12:39
466,11:43
294,17:40
480,06:58
578,20:42
93,01:21
519,19:20
321,13:55
275,18:25
85,21:57
42,16:48
68,07:52
458,00:40
572,16:16
62,14:05
271,11:38
158,03:40
322,21:46
381,11:59
1,16:43
17,14:03
240,14:09
465,02:26
212,04:25
593,19:49
503,08:11
493,19:48
481,12:51
295,02:16
507,19:52
196,15:16
494,11:54
279,19:37
349,23:07
472,14:53
335,05:08
169,12:01
306,08:07
72,22:19
518,06:49
261,21:06
175,13:35
484,05:29
401,06:53
374,11:02
9,23:11
351,13:52
240,07:20
248,19:41
14,16:12
291,23:09
551,01:05
37,08:10
217,05:33
40,07:43
266,13:23
518,05:36
409,13:14
521,16:15
86,01:34
476,23:43
552,15:45
140,21:50
59,09:38
463,11:01
63,01:54
106,07:21
326,20:01
383,02:56
362,07:21
572,03:40
570,03:29
15,16:20
423,18:36
58,12:43
136,15:28
120,04:36
304,18:24
112,14:01
476,20:42
460,12:57
179,18:42
485,08:04
438,15:29
314,05:41
256,22:40
332,14:17
451,17:15
603,12:20
79,17:36
403,20:51
85,10:05
99,18:20
319,12:02
29,06:20
156,06:24
17,20:38
400,05:55
177,06:26
489,19:22
359,15:00
379,03:41
569,11:56
555,02:16
308,12:31
401,07:41
506,03:29
18,10:46
535,20:28
409,10:03
447,21:34
98,03:39
357,11:00
411,11:25
601,08:16
361,20:36
96,14:38
456,23:28
76,20:58
194,12:12
14,05:58
129,02:27
521,01:40
282,01:12
395,02:55
303,23:11
184,02:54
2,17:24
106,01:06
68,19:03
204,08:00
554,08:04
84,12:34
369,14:40
420,01:13
57,06:18
565,08:56
343,21:05
156,15:25
162,01:29
222,22:56
65,19:34
312,23:34
214,07:56
253,06:03
37,09:46
447,17:39
467,07:39
134,11:07
546,00:59
328,22:56
180,13:35
73,00:39
283,09:02
337,01:27
549,13:24
337,16:29
10,07:32
266,14:44
219,03:13
514,03:16
133,11:47
339,00:10
549,05:02
253,19:45
275,07:06
124,12:00
505,17:20
462,04:24
490,17:14
279,12:35
95,05:26
503,12:14
219,23:54
15,04:22
129,04:37
154,21:59
157,00:44
249,01:06
402,03:09
114,07:07
518,08:37
446,19:35
472,10:46
11,21:35
115,06:11
269,01:26
474,21:23
400,12:19
307,21:11
87,21:12
594,12:43
505,13:39
151,07:41
500,13:20
386,09:59
100,06:23
54,02:52
341,04:23
123,19:48
486,07:32
360,07:45
28,00:53
254,06:17
531,00:13
105,15:21
129,15:08
545,16:54
262,23:09
590,19:09
583,08:17
234,05:27
166,12:51
237,19:07
262,11:14
531,17:21
193,04:15
450,21:13
328,17:16
382,12:41
15,21:29
441,18:46
348,22:13
76,01:16
464,16:40
203,18:36
404,06:20
590,22:38
391,18:32
212,03:35
217,16:10
575,17:48
93,11:35
113,03:10
380,21:49
595,01:37
553,15:44
181,23:45
50,15:02
269,19:51
156,14:54
402,02:34
293,22:04
470,16:13
486,23:43
356,22:52
154,15:56
410,05:16
204,04:58
128,02:54
516,19:07
548,22:27
394,21:29
429,06:13
326,05:38
19,15:58
459,10:36
426,09:59
41,23:16
365,21:20
478,22:28
48,03:26
568,16:16
571,01:54
39,07:17
90,11:19
584,05:38
348,19:42
73,19:24
468,15:04
530,21:40
385,17:53
327,18:09
477,05:00
332,13:46
438,10:25
226,04:58
88,12:28
409,06:48
224,16:08
179,00:01
449,09:49
61,11:57
90,22:05
330,05:57
107,15:36
43,08:45
404,17:36
56,17:29
558,12:13
365,11:53
197,20:44
104,20:42
36,21:56
592,09:32
511,11:28
370,14:56
271,19:22
43,01:19
436,20:23
393,05:29
91,03:43
459,10:17
531,07:24
215,02:19
470,19:36
71,07:41
243,07:20
89,10:56
528,07:47
303,00:03
365,20:11
489,19:57
294,16:01
184,05:09
535,21:04
46,22:57
198,03:46
70,18:09
62,14:41
533,13:52
572,06:48
459,21:17
32,05:04
544,02:45
378,21:29
261,12:40
313,20:35
253,16:22
476,15:41
438,00:27
376,14:08
224,07:25
449,15:30
294,20:04
75,03:22
477,02:51
540,20:41
375,17:44
457,01:32
76,19:56
375,14:11
257,02:21
205,05:37
209,02:28
292,23:26
502,10:44
517,18:21
451,06:36
20,17:51
61,20:38
597,11:31
479,21:58
158,13:41
128,00:26
479,01:18
360,17:59
272,07:40
271,11:29
242,17:01
272,19:13
304,14:35
131,16:10
247,08:55
563,14:28
95,03:27
518,05:44
235,17:55
68,21:34
317,20:11
401,09:23
363,01:14
552,15:57
64,21:55
525,21:53
474,14:24
177,01:26
144,08:24
419,03:27
256,14:36
418,04:56
312,20:39
51,03:06
83,01:06
97,17:56
281,15:54
481,12:37
21,08:35
211,09:16
324,16:32
58,07:27
444,08:54
66,20:55
177,06:31
141,11:36
271,13:10
17,11:52
349,20:27
64,14:10
411,14:39
90,21:03
49,19:19
340,00:00
540,13:10
542,09:28
586,16:52
508,05:47
316,20:27
225,04:41
142,09:44
320,03:36
264,01:24
510,01:21
451,13:24
40,23:29
427,02:37
576,21:26
344,18:55
161,07:12
343,20:20
438,21:21
162,20:23
447,21:35
509,18:16
542,06:14
232,12:32
208,18:00
182,02:27
435,23:09
130,08:22
464,16:49
342,19:56
442,19:19
487,04:58
22,11:57
178,06:13
465,05:17
86,23:10
282,01:00
505,07:38
541,03:55
21,20:02
186,16:20
292,19:45
488,01:03
10,04:21
366,20:42
347,18:40
1,01:22
317,16:37
547,21:19
64,20:52
266,00:33
308,12:23
108,22:03
70,15:34
487,21:22
291,04:02
93,15:11
451,19:31
41,10:53
601,22:53
138,22:18
155,23:35
176,07:19
500,10:12
493,09:35
124,23:11
523,01:55
305,09:40
79,07:25
185,07:41
181,22:00
48,11:25
457,06:52
450,00:20
329,20:16
455,23:08
202,17:53
586,19:11
370,22:20
337,03:36
181,12:54
504,21:11
243,03:37
