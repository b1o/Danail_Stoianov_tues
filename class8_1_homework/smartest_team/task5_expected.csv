17,20:38
21,20:02
39,20:38
61,20:38
64,20:52
66,20:55
76,20:58
104,20:42
122,20:23
122,20:09
142,20:42
150,20:54
151,20:59
162,20:23
197,20:44
217,20:09
241,20:35
255,20:16
294,20:04
312,20:39
313,20:35
316,20:27
317,20:11
326,20:01
329,20:16
343,20:20
349,20:27
361,20:36
365,20:11
366,20:42
389,20:22
403,20:51
436,20:23
450,20:04
476,20:42
511,20:21
535,20:28
540,20:41
540,20:57
549,20:18
578,20:42
582,20:52
594,20:40
602,20:31
